Sources/Tracking/CustomerIO.swift

```diff
@@ -231,7 +228,7 @@ public class CustomerIO: CustomerIOInstance {
     /**
      Sets credentials on shared or non-shared instance.
      */
-    internal func setCredentials(siteId: String, apiKey: String, region: Region, diGraph: DIGraph) {
+    internal func setCredentials(siteId: String, apiKey: String, region: Region) {
         var credentialsStore = diGraph.sdkCredentialsStore
```

Comment:
looks like `diGraph` was removed from the params so does this not need to be `DIGraph.getInstance`?